add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -595,6 +595,27 @@ var LoopRoomsInternal = {
   },
 
   /**
+   * Request information about number of participants joined to a specific room from the server.
+   * Used to determine infobar message state on the number of participants in the room.
+   *
+   * @param {String}  roomToken Room identifier
+   * @return {Number} Count of participants in the room.
+   */
+  getNumParticipants: function(roomToken) {
+    try {
+      if (this.rooms && this.rooms.has(roomToken)) {
+        return this.rooms.get(roomToken).participants.length;
+      }
+      return 0;
+    }
+    catch (ex) {
+      // No room, log error and send back 0 to indicate none in room.
+      // throw new Error("No room found in current session: ", ex);
+      return 0;
+    }
```

Comment:
This reasoning seems fine; I just uncommented the throw statement and turned it into an error.